virttest/libvirt_xml/vm_xml.py

```diff
@@ -169,7 +169,7 @@ class VMXMLBase(base.LibvirtXMLBase):
                  'current_vcpu', 'vcpus', 'os', 'cpu', 'pm', 'on_poweroff',
                  'on_reboot', 'on_crash', 'features', 'mb', 'max_mem_unit',
                  'current_mem_unit', 'memtune', 'max_mem_rt', 'max_mem_rt_unit',
-                 'max_mem_rt_slots', 'iothreads', 'iothreadids', 'memory')
+                 'max_mem_rt_slots', 'iothreads', 'iothreadids', 'memory', 'perf')
```

Comment:
commit message does not have mention about `perf` tags, pls include and little more detailed commit message would be helpful

Reply:
The perf part has been merged. There is a version issue,